gce-containers-startup/utils/registry.go

```diff
@@ -16,29 +16,17 @@ package utils
 
 import "strings"
 
-// isDefaultRegistryMatch determines whether the given image will
-// pull from the default registry (DockerHub) based on the
-// characteristics of its name.
-func IsDefaultRegistryMatch(image string) bool {
+// Returns true iff the provided image string points to repository
+// that uses a GCP token. Currently, that is:
+//  - gcr.io
+func UseGcpTokenForImage(image string) bool {
 	parts := strings.SplitN(image, "/", 2)
 
 	if len(parts[0]) == 0 {
 		return false
 	}
 
```

Comment:
You could also return false if len(parts) < 2, that is there is no hostname. The code works anyway, but you're then calling hostname something that is not hostname, sounds error prone.

Reply:
Done.